src/main/java/jenkins/advancedqueue/JobGroup.java

```diff
@@ -181,16 +240,29 @@ public boolean isUsePriorityStrategies() {
 		return usePriorityStrategies;
 	}
 
+	@DataBoundSetter
 	public void setUsePriorityStrategies(boolean usePriorityStrategies) {
 		this.usePriorityStrategies = usePriorityStrategies;
 	}
 
-	public List<JobGroup.PriorityStrategyHolder> getPriorityStrategies() {
+	public List<PriorityStrategy> getPriorityStrategies() {
 		return priorityStrategies;
 	}
 
-	public void setPriorityStrategies(List<JobGroup.PriorityStrategyHolder> priorityStrategies) {
-		this.priorityStrategies = priorityStrategies;
+	@DataBoundSetter
+	public void setPriorityStrategies(List<? extends PriorityStrategy> priorityStrategies) {
+		if (priorityStrategies.get(0) instanceof PriorityStrategyHolder) {
+			this.priorityStrategies = convertPriorityStrategyHolder((List<PriorityStrategyHolder>)priorityStrategies);
+		} else {
+			this.priorityStrategies = (List<PriorityStrategy>) priorityStrategies;
+		}
+	}
+
+	public List<PriorityStrategy> convertPriorityStrategyHolder(List<PriorityStrategyHolder> priorityStrategyHolders) {
+		List<PriorityStrategy> B = priorityStrategyHolders.stream()
+        .map(holder -> holder.priorityStrategy)
+        .collect(Collectors.toList());
+		return B;
 	}
 
 	/**
```

Comment:
Most likely, `newInstance()` is no longer required,unless there are internal dependencies on this deprecated databinding mechanism

Reply:
Will keep newInstance and do a Refactoring to the new DataBound mechanism later.
Can you point me to a good documentation or example code of it.